### DISCUSSION TITLE
Is it possible to change splash screen on BW radios?

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#4923] [Q&A] [answered]
I looked on the RadioMaster MT12 memory card and I can't find any splash screen images anywhere. Is it possible to add a custom PNG as a startup screen?

**Comment by pfeerick** [comment, marked as answer]
Yes, you can set a splash screen on B&W radios, but it must be done using
Companion as the firmware is modified to change the splash screen embedded
inside it. Due to B&W radios generally having less RAM than colorlcd
radios.

This should still be mostly relevant : https://youtu.be/E2RO-YWRIIU

**Reply by p0mp0wnik** [reply]
The only problem is the fact that edgetx comapnion hasn't supported mt12 yet so it can't flash changed software. However some guy on YouTube has changed his splash screen but hasn't showed how to

**Comment by pfeerick** [comment]
Which simply means he is using a self-build or an automated build of
Companion from main (i.e. the dev version of 2.11) which *does* support the
MT12 ;)

https://github.com/EdgeTX/edgetx/actions/workflows/win_cpn-64.yml?query=branch%3Amain

https://github.com/EdgeTX/edgetx/actions/workflows/macosx_cpn.yml?query=branch%3Amain

https://github.com/EdgeTX/edgetx/actions/workflows/linux_cpn.yml?query=branch%3Amain
